recipes/pyrfr/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pyrfr" %}
+{% set version = "0.8.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 092cb7931486672ad00a71f37cc2e204cdb4b75ab0a47056667cc2683cc07dbd
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - boost
+    - {{ compiler('cxx') }}
+    - doxygen
+    - swig >3.0
```

Comment:
Could you please make this more strict and specify that this can't be SWIG 4 or higher; and there's also a comment by a user that there's a minimal SWIG version that has to be installed: https://github.com/automl/random_forest_run/issues/52

Reply:
Has been added, thanks for the heads up.